raster-foundry/raster-foundry
# Upgrade doobie to 0.7.0

## Overview
Upgrade doobie to 0.7.0

### Checklist

- ~Swagger specification updated~
- ~New tables and queries have appropriate indices added~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

### Notes
No breaking changes affected us

## Testing Instructions

- assemble backsplash / api
- run tests
- browse the app and verify that database stuff works still

Closes #794
